### ISSUE TITLE
Open with periodicity "n" within a range of consecutive monthdays

### ISSUE THREAD

**Issue by zetx16** [opening-hours/opening_hours.js#252]
Open with periodicity "n" within a range of consecutive monthdays, the range is followed by "/n" (e.g., > 02-30/n)
Using in signs.
odd: `01-31/2`
even: `02-30/2` 
![image](https://user-images.githubusercontent.com/8606466/36323341-a407d214-1361-11e8-803a-57e0f6ae11b7.png)

**Comment by ypid** [comment]
This is currently [not specified](https://wiki.openstreetmap.org/wiki/Key:opening_hours/specification#selector:month). Opening an issue downstream (here against opening_hours.js) is not the right way to go since I only implement the specification. Please go though the [Proposal process](https://wiki.openstreetmap.org/wiki/Proposal_process) and describe your use case in greater detail because your initial information is a bit rare for me to understand it. Please close the issue for now. You can reopen it once/if your proposal is approved.

**Comment by zetx16** [comment]
Usage discussed [here](https://wiki.openstreetmap.org/wiki/Talk:Key:opening_hours#odd.2Feven_dates). And in [opening_hours wiki page](https://wiki.openstreetmap.org/wiki/Key:opening_hours#Rules) described (fourth in list).

**Comment by ypid** [comment]
Thanks for the input. You are right. Ref to the original spec: https://openingh.ypid.de/netzwolf_mirror/time_domain/specification.html#rule5 So I would say this is already specified and should also be implemented in here then.

**Comment by mnalis** [comment]
Interestingly enough, it does seem to **work** in current [evaluation tool](https://openingh.openstreetmap.de/evaluation_tool/) **if single month is specified**; for example:

`Jan 1-31/2: Mo-Fr 08:00-13:00; PH off` seem to work OK for only odd dates in January (Monday-Friday only)! 

However, trying to specify it for all year like this: `Jan-Dec 1-31/2: Mo-Fr 08:00-13:00; PH off`  fails to parse. Also `Jan,Feb,Mar,Apr,May,Jun,Jul,Aug,Sep,Oct,Nov,Dec 1-31/2: Mo-Fr 08:00-13:00; PH off`  seems to parse ok, but does something totally unexpected)

**Comment by mnalis** [comment]
Also, [OSM Wiki for Key:opening_hours](https://wiki.openstreetmap.org/wiki/Key:opening_hours) specifies the rule:
> Open with periodicity "n" within a range of consecutive monthdays, the range is followed by "/n" (e.g., > 02-30/n)

[Formal specification](https://wiki.openstreetmap.org/wiki/Key:opening_hours/specification#Known_issues) links back to this issue as "Known issue", so it appears it waits for a fix in opening_hours.js first. I think it should be implemented here, so the _known issue_ in current formal specification can be  replaced with description and example of usage.

**Comment by ypid** [comment]
I agree with you. Needs to be fixed in opening_hours.js and in the OSM Wiki.

**Comment by rhhsm** [comment]
Hoping this will be implemented soon too! I'm trying to evaluate if "1-31/2 12:00-17:00; 2-30/2 10:00-20:00; 2-30/2 Sa 11:00-18:00; 1-31/2 Sa off; Su off" is valid but "/" is apparently not supported. Lots of doctors here in Bulgaria have working hours that depend on the odd/even-ness of the date of the month.
![20200905_122149](https://user-images.githubusercontent.com/69100427/92328830-27051480-f06c-11ea-89e0-ab8be7690f0a.jpg)

**Comment by ypid** [comment]
Hi @rhhsm, please just use https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments to signal your interest. Also, you may want to refer to: #341
